pyk/src/pyk/kdist/__main__.py

```diff
@@ -48,13 +48,15 @@ def _exec_build(
     force: bool,
     verbose: bool,
     debug: bool,
+    clean: bool,
```

Comment:
I wonder, would it make sense to make `clean=False` behavior the only option? After all, if a build step requires an empty output directory, it can ensure that as the first step of the build.

Reply:
> this change is a middle ground that we can get in for now to the people who requested it

Can you confirm if this solution acceptable for them? If so, I fine with this workaround.

Reply:
I confirmed that this will work.